test/models/search_facet_test.rb

```diff
@@ -0,0 +1,99 @@
+require "test_helper"
+
+class SearchFacetTest < ActiveSupport::TestCase
+
+  def setup
+    @search_facet = SearchFacet.first
+
+    # filter_results to return from mock call to BigQuery
+    @filter_results = [
+        {id: 'NCBITaxon_9606', name: 'Homo sapiens'},
+        {id: 'NCBITaxon_10090', name: 'Mus musculus'}
+    ]
+  end
+
+  # should return expected filters list
+  # mocks call to BigQuery to avoid unnecessary overhead
+  test 'should update filters list' do
+    puts "#{File.basename(__FILE__)}: #{self.method_name}"
+
+    # mock call to BQ, until a better library can be found
+    mock = Minitest::Mock.new
+    mock.expect :query, @filter_results, [String]
+
+    SearchFacet.stub :big_query_dataset, mock do
+      filters = @search_facet.get_unique_filter_values
+      assert_equal @filter_results, filters
+      mock.verify
+    end
+
+    puts "#{File.basename(__FILE__)}: #{self.method_name} complete!"
```

Comment:
Are these puts necessary?  I would expect that whatever test running library we would use would have a --verbose option to print stuff like this

Reply:
This is something that I do all over the place... I just like seeing which tests are being run and from where in the output (either in the console or in CI).  Let's chat about this because if we take them out, there's about a dozen other test suites that need to get changed.

Reply:
good idea, it's a larger convo